docs/development/ipv6.md

```diff
@@ -7,8 +7,6 @@
 [GEP-21](../proposals/21-ipv6-singlestack-local.md) proposes IPv6 Single-Stack Support in the local Gardener environment.
 This documentation will be enhanced while implementing GEP-21, see [gardener/gardener#7051](https://github.com/gardener/gardener/issues/7051).
 
-To use IPv6 single-stack networking, the [feature gate](../deployment/feature_gates.md) `IPv6SingleStack` must be enabled on gardener-apiserver and gardenlet.
```

Comment:
> Infrastructure-specific validations will be added in parallel to the corresponding provider extensions.

Can you add this information to the release note?

Reply:
I added the sentence to the release note.

Is that what you wanted or would you like a second release note instead?

Reply:
Thanks. Sounds good 👍